detect_variants/detect_variants.cwl

```diff
@@ -36,9 +36,11 @@ inputs:
          type: File
          secondaryFiles: [.tbi]
     vep_cache_dir:
-        type: File
+        type: string?
```

Comment:
Nice, so it validates as a string, but it just so happens to be a Directory(which is not supported).  Works for me!